src/tiles.jl

```diff
@@ -0,0 +1,105 @@
+# This file is a work in progress...
+
+"""
+    WindowedProblem(op; size, centers, θ)
+
+Combine multiple compute operations into a single object, 
+to be run over the same windowed grids.
+
+"""
+@kwdef struct WindowedProblem{P,S,WC} <: AbstractProblem
```

Comment:
Couldn't we generalise a tile and a window by having a window with a buffer, to be determined by the user? A tile would be a window with a null buffer. Also, could we ask the user to provide target pixels, that could define the core window?

Reply:
Just discussed this with @ninbrm and the key difference is really that `TiledWindow` stores files as it goes for long computes. So I'll rename it to `StoredWindowProblem` or something and let it hold either `WindowedProblem` or just `Problem` so you can control the nesting

Reply:
And yes probably allowing a matrix of all the target pixels would be useful